### PR TITLE
Group support needed configuration (org_metadata_api) and tasks

### LINKED ISSUE #5244: Document Groups configuration steps at NEWS
- [X] Add needed information at NEWS.
- [X] Create a rake task that sets CDB_Conf for groups api.

### PR DESCRIPTION
**Pull request by juanignaciosl**
This closes #5244 and will be merged into #4924 branch. @luisbosque (since it involves configuration files) and @viddo (since you've just had to configure it), could you CR this, please?

@luisbosque Groups are expected to be merged into master in a few days (this or next week). Needed operation work is being documented here: CartoDB/cartodb-platform/issues/993. No need to begin yet, but keep this in mind, I'll soon be asking you about this :-)

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -3,7 +3,11 @@
 -------------------
 * Synchronizations model now has a new field (and FK) to visualizations.id and joins to them using that instead of by matching name to canonical visualization's table name. It also gets deleted if FK dissapears.
 * Code also switches to using syncrhonizations.visualization_id for linking, so in order to have back existing synchronizations, the following rake needs to be run: `bundle exec rake cartodb:populate_synchronization_visualization_ids`
-* Group support for organizations. Note: in order to use groups you need to run the server in multithread mode: `bundle exec thin start --threaded -p 3000`.
+* Group support for organizations. Usage instructions:
+  1. Update CartoDB PostgreSQL extension to the latest version ([instructions](https://github.com/CartoDB/cartodb/blob/master/UPGRADE#L43)).
+  2. Trigger configuration: `RAILS_ENV=development bundle exec rake cartodb:db:configure_extension_org_metadata_api_endpoint`.
+  3. Trigger org owner role assignment: `RAILS_ENV=development bundle exec rake cartodb:db:assign_org_owner_role`.
+  4. From now on you must run the server in multithread mode: `bundle exec thin start --threaded -p 3000 --threadpool-size 5`.
```
Comment (viddo) [inline NEWS.md]: Isn't the app_config change missing? The `sync_db_api` key

Also, I presume the new rake tasks do the steps I had to do manually in the postgresql console, right?

Comment (juanignaciosl) [inline NEWS.md]: I have renamed it to `org_metadata_api`. "Sync" is too overloaded and would lead to confusion.

Comment (viddo) [inline NEWS.md]: Ok, that sounds better :+1:, but my question was if the new config setting shouldn't be mentioned here, in the NEWS file?

Comment (juanignaciosl) [inline NEWS.md]: Oh, you're right, thanks!

**Comment by juanignaciosl** [comment]
Retest this, please.

**Pull request merged**